v1/metrics.yaml

```diff
@@ -429,115 +493,1791 @@ paths:
                 x-client-ip: '{{x-client-ip}}'
       x-monitor: false
 
-definitions:
-  listing:
-    description: The result format for listings
-    required:
-      - items
-    properties:
-      items:
-        type: array
-        items:
-          type: string
+  ########################################
+  # Edited Pages
+  ########################################
+  /edited-pages/:
+    get:
```

Comment:
`x-request-handler` or `operationId` are missing here.

Reply:
hm, those endpoints are for documentation only (/pageviews/ one). Should we remove all of them?

Reply:
Yes, because they don't provide any functionality and the documentation that is present is short enough to be folded into end points that actually exist. On that note, each end point in this YAML should have the stability and licence bullet points.

Reply:
Ok, no-data endpoints removed. Also all endpoints have license and stability points.